src/components/Search.tsx

```diff
@@ -2,8 +2,44 @@ import React, { Component } from 'react';
 import { InputAdornment, TextField } from '@material-ui/core';
 import SearchIcon from '@material-ui/icons/Search';
 import { Redirect } from 'react-router';
+import styled from 'styled-components';
 
 import { SEARCH_PARAM_QUERY } from '../constants';
+import { colors, fonts, ScreenReaderOnly } from '../App.styles';
+
+const SearchInput = styled(TextField)`
+  && {
+    width: 100%;
+    background: ${colors.white};
+    color: ${colors.black};
+  }
+  && > div {
+    margin-top: 0;
+  }
+  label {
+    ${ScreenReaderOnly}
+  }
+  input {
+    padding: 7px 7px 7px 0;
```

Comment:
Any reason we're mixing pixels and `em`s? [`rem`](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/) ftw

Reply:
The pixels were just added to make testing styling easier. I'll create a separate PR that implements a helper function to convert pixel values to rems and implement where necessary.